src/domain/aboutus/components/heading_section/HeadingSection.jsx

```diff
@@ -1,28 +1,45 @@
 import React from 'react';
 import classes from './HeadingSection.module.scss';
 import { Text } from '../../../../components';
-import { aboutUs } from '../../../../assets';
+// import { aboutUs } from '../../../../assets';
```

Comment:
I dont think these comments need to be here anymore as we arent using any of the assets
```suggestion
```

Reply:
I apologize, these comments are a bad habit of my lack of experience. I am new to branching.